README.rst

```diff
@@ -162,6 +162,67 @@ To understand the details of this statement, take a look at the code of
 `django_loci.storage.OverwriteStorage
 <https://github.com/openwisp/django-loci/blob/master/django_loci/storage.py>`_.
 
+``DJANGO_LOCI_GEOCODER``
+~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``str``     |
++--------------+-------------+
+| **default**: | ``ArcGIS``  |
++--------------+-------------+
+
+Service used for geocoding and reverse geocoding.
+
+Supported geolocation services:
+
+* `ArcGIS`
+* `Nominatim`
+* `GoogleV3` (Google Maps v3)
+
+``DJANGO_LOCI_GEOCODE_STRICT_TEST``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-----------+
+| **type**:    | ``bool``  |
++--------------+-----------+
+| **default**: | ``True``  |
++--------------+-----------+
+
+If geocoding tests failed and set to ``False`` won't raise error.
```

Comment:
Change to:

```
Indicates whether the system should raise an ``ImproperlyConfigured``
exception in case geocoding doesn't work when the application is started.
```